index.ts

```diff
@@ -20,13 +20,17 @@ export interface FeatureData {
     usage_stats?: usage_stats_url | [usage_stats_url, usage_stats_url, ...usage_stats_url[]];  // A single URL or an array of two or more
 }
 
-type browserIdentifier = "chrome" | "edge" | "firefox" | "safari";
+type browserIdentifier = "chrome" | "chrome_android" | "edge" | "firefox" | "firefox_android" | "safari" | "safari_ios";
 
 interface SupportStatus {
-    /** Whether the feature achieved baseline status */
+    /** Whether the feature achieved Baseline status */
     is_baseline: boolean;
-    /** Date the feature achieved baseline status */
+    /** Whether the feature is widely supported (high) or newly supported (low) */
+    substatus?: "high" | "low";
+    /** Date the feature achieved Baseline status (legacy) */
     since?: string;
+    /** Date the feature achieved Baseline low status */
+    keystone_date?: string;
```

Comment:
If we can't just keep `since` above, I think `baseline_date` or `baseline_since` would be better here than `keystone_date`. We use that terminology in the definition to define what is Baseline, but from the outside in, it makes just as much or more sense to talk about a Baseline date, I think.